antora-playbook.yml

```diff
@@ -79,7 +79,12 @@ content:
       - '!v2.0.0.alpha0' # same SNAPSHOT version as another branch
       - '!v1.1.1' # Tag v1.1.1 has version number 1.1.1-SNAPSHOT use v1.1.1-doc instead
       - '!v1.1.1-rebuild' # I thought I needed to remake the debs, but I did not
-  - url: https://github.com/OpenNMS/opennms-provisioning-integration-server.git
+    - url: https://github.com/OpenNMS/opennms-velocloud-plugin
+    start_path: docs
+    branches: main
```

Comment:
He made a branch named "release-1.x" you need to reference it here instead of "main". It seems to me he doesn't want or just simply can't make a release. What will happen the version number will be shown as [1.0.0-SNAPSHOT](https://github.com/OpenNMS/opennms-velocloud-plugin/blob/99792066c0ed0adb49ac37c9bd105cae415e3d42/docs/antora.yml#L2). If this is ok you can for it.

Reply:
For sustainability, I don't think that will work, as we need to make sure the Velocloud docs update with each major release as well. 
 
If I reference `release-*` will that work? Or is there something @christianpape needs to do so we can set up the Antora syntax correctly?